lib/utils.js

```diff
@@ -265,16 +265,16 @@ var sendRequest = function(options, args, connectionInfo, callback, retryCounter
   return request(options)
     .then(successHandler)
     .catch(ConnectError, function(e) {
-      log.error('A ConnectError happened: \n' + e);
+      log.error( { message: 'A ConnectError happened', error: e } );
       return retryWhenConnectionFails(args, connectionInfo, callback, retryCounter);
       // TODO: how to prevent a throw or rejection from also stepping thru the other catch-blocks?
     })
     .catch(RateLimitingError, function(e) {
-      log.error('A RateLimitingError error like "429 Too Many Requests" happened: \n'
-        + 'statusCode: ' + e.statusCode + '\n'
-        + 'body: ' + e.response.body + '\n'
+      log.error( { message: 'A RateLimitingError error like "429 Too Many Requests" happened',
+        statusCode: e.statusCode,
+        body: e.response.body
```

Comment:
It will be nice to wrap the error for will:
```
logger.error({message: 'A RateLimitingError error like "429 Too Many Requests" happened',
   error: {
       statusCode: e.statusCode,
       body: e.response.body
    }
})
```